pages/docs/community/styleguide/code-examples.md

```diff
@@ -0,0 +1,49 @@
+# Code Examples
+
+## Guidelines
+
+Code examples which you add must be simple enough to understand and with good details to do something interesting and useful.
+
+You need to add pratical code examples which adheres to industry standards as users will copy the sample code and run it in production.Be sure to include any warnings thriugh comments which might occur during code run.
+Make it clear if it is a snippet and not a full example.Provide information about any packages or dependencies which would be required to run the sample code.
+
+The code example should be self explanatory and easy to understand. The goal is to provide simplified working examples that can be understood as soon as possible, not necessarily to create efficient, brilliant code that impresses experts and has outstanding utility.
+
+
+## Formatting 
+
+### Code Length
+
+- Code lines shouldn't be so lengthy that reading them horizontally requires scrolling.
+
+- For the sake of readability, break large lines at appropriate moments, but never at the price of acceptable practises.
+
+- Add bullet points for better readability and understandability.
+
+
+### Colors
+
+Using different colors for making the code more readable and understandable.
+Add different color to `tags` and `code` is a good practice.
+
+Code blocks that show good habits to adopt are shown with a green check mark in the right corner, while those that show bad habits are shown with a white cross inside a red circle.
+
+You different colors for bracket to know the starting and ending bracket of code block.
+
+### Spacing and Indentation
+
+Put the characters for the open-brace ("") on the same line as the sentence that opens the block.
+
+In all code samples, the indentation should be two spaces per tab.]
+
+Add proper spacing between code blocks
+
+After adding comments ,start writing code from the next line.
+## General
```

Comment:
```suggestion
### General
```
be consistent with the headings. # for main, ## & ### for subheadings and should be mainly used.